### PR TITLE
sst_importer: add wire extra bytes into the packed size

### LINKED ISSUE #14313: pitr restore data fail for "failed: raft entry is too large, region 170, entry size 8616655"
### What version of TiKV are you using?

2.28 nightly
### Steps to reproduce
start log backup
do full backup every sometime
run paypay workload
do pitr restore with latest full backup repeatedly (clean target TiDB eachtime)

### What did you expect?
pitr backup and restore works normal
### What did happened?
```code
failed: raft entry is too large, region 170, entry size 8616655; execute over region id:170 start_key:\"t\\200\\
000\\000\\000\\000\\000\\000\\377e_i\\200\\000\\000\\000\\000\\377\\000\\000\\001\\003\\200\\000\\000\\000\\377\\000
\\031\\014\\336\\000\\000\\000\\000\\373\" end_key:\"t\\200\\000\\000\\000\\000\\000\\000\\377e_i\\200\\000\\000\\00
0\\000\\377\\000\\000\\001\\003\\200\\000\\000\\000\\377\\000\\035O\\237\\000\\000\\000\\000\\373\" region_epoch:<co
nf_ver:1 version:158 > peers:<id:171 store_id:1 >  failed: raft entry is too large, region 170, entry size 8616655;
execute over region id:170 start_key:\"t\\200\\000\\000\\000\\000\\000\\000\\377e_i\\200\\000\\000\\000\\000\\377\\0
00\\000\\001\\003\\200\\000\\000\\000\\377\\000\\031\\014\\336\\000\\000\\000\\000\\373\" end_key:\"t\\200\\000\\000
\\000\\000\\000\\000\\377e_i\\200\\000\\000\\000\\000\\377\\000\\000\\001\\003\\200\\000\\000\\000\\377\\000\\035O\\
237\\000\\000\\000\\000\\373\" region_epoch:<conf_ver:1 version:158 > peers:<id:171 store_id:1 >  failed: raft entry
 is too large, region 170, entry size 8616655; execute over region id:170 start_key:\"t\\200\\000\\000\\000\\000\\00
0\\000\\377e_i\\200\\000\\000\\000\\000\\377\\000\\000\\001\\003\\200\\000\\000\\000\\377\\000\\031\\014\\336\\000\\
000\\000\\000\\373\" end_key:\"t\\200\\000\\000\\000\\000\\000\\000\\377e_i\\200\\000\\000\\000\\000\\377\\000\\000\
\001\\003\\200\\000\\000\\000\\377\\000\\035O\\237\\000\\000\\000\\000\\373\" region_epoch:<conf_ver:1 version:158 >
 peers:<id:171 store_id:1 >  failed: raft entry is too large, region 170, entry size 8616655; execute over region id
:170 start_key:\"t\\200\\000\\000\\000\\000\\000\\000\\377e_i\\200\\000\\000\\000\\000\\377\\000\\000\\001\\003\\200
\\000\\000\\000\\377\\000\\031\\014\\336\\000\\000\\000\\000\\373\" end_key:\"t\\200\\000\\000\\000\\000\\000\\000\\
377e_i\\200\\000\\000\\000\\000\\377\\000\\000\\001\\003\\200\\000\\000\\000\\377\\000\\035O\\237\\000\\000\\000\\00
0\\373\" region_epoch:<conf_ver:1 version:158 > peers:<id:171 store_id:1 >  failed: raft entry is too large, region
170, entry size 8616655\n", "error": "command terminated with exit code 1"} 
```

### PR DESCRIPTION
**Pull request by YuJuncen**
### What is changed and how it works?

Issue Number: Close #14313

What's Changed:
This PR added some extra bytes for each modification, so the `unpacked_size` would be nearer to the real raft entry size.

```commit-message
```

### Check List 

Tests 

- Manual test (add detailed scripts or steps below)

I have tested it in the internal cluster, if you are a PingCAP employee, you can check it at [here](https://tcms.pingcap.net/dashboard/executions/case/3878188). There aren't suitable screenshots to be published, sorry for that.

And I guess this can also be checked by any table with average row size less than 70B (see the comments), but for now I don't have suitable cluster for running such tests...

### Release note 

```release-note
None
```
(This is a fix over the master branch.)

### CODE REVIEW

File: src/import/sst_service.rs [hunk 1]
```diff
@@ -104,6 +117,19 @@ struct RequestCollector {
 }
 
 impl RequestCollector {
+    fn record_message_of_size(&mut self, size: usize) {
```
Comment (tonyxuqqi) [inline src/import/sst_service.rs:120]: minor: record_size_of_message or record_size seems better to me.

**Review by tonyxuqqi — APPROVED** [review]

**Review by tabokie — APPROVED** [review]

**Comment by tabokie** [comment]
Better add a test.

**Comment by BusyJay** [comment]
Maybe we should let the engine to tell the size of modification. In raftkv2, writes are encoded using more simple codec and can batch up more modifications than v1.

**Comment by YuJuncen** [comment]
@BusyJay I agree the idea of letting the engine provide some way to calculate "real size" of some write, and I think before we figuring out some ideal interface definition, perhaps we can merge this for solving the problem we have encountered now?

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**